src/pages/index.js

```diff
@@ -1,48 +1,32 @@
-import { enableValidation, settings } from "../scripts/validation.js";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
 import "./index.css";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
 
 //profile pics
 import logoSrc from "../images/Logo.svg";
 import avatarSrc from "../images/avatar.jpg";
 import editIconSrc from "../images/Edit-Icon.svg";
 import plusIconSrc from "../images/Plus-icon.svg";
+import avatarEditIconSrc from "../images/white-edit-avatar.svg";
+
+//avatar modal
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarBtn = document.querySelector(".profile__avatar-btn");
+const avatarCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarForm = avatarModal.querySelector("#edit-avatar-form");
 
 document.getElementById("header-logo").src = logoSrc;
 document.getElementById("profile-avatar").src = avatarSrc;
```

Comment:
`document.getElementById("profile-avatar")`

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine